common/views/components/RelatedArchiveWorks/RelatedArchiveWorks.js

```diff
@@ -0,0 +1,198 @@
+// @flow
+import { type Work } from '@weco/common/model/work';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import NextLink from 'next/link';
+import fetch from 'isomorphic-unfetch';
+import { classNames, cssGrid, font } from '../../../utils/classnames';
+import { workLink } from '@weco/common/services/catalogue/routes';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
+import Space from '@weco/common/views/components/styled/Space';
+
+type Collection = {|
+  path: {|
+    path: string,
+    level: string,
+    label: string,
+    type: string,
+  |},
+  work: {|
```

Comment:
Could we not just expand on the current work model to gain the benefits everywhere?

Reply:
Not sure I follow, do you mean add this Collection type to the Work type or use the Work type in this Collection type. If the latter the work models are different, no?

Reply:
Nope, the work property is a `Work`. Just wondering if we start filling out that work type finally as [it's an `Object` currently](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/model/work.js)